src/arch/arm/armmlib/exception_handlers.c

```diff
@@ -16,14 +16,30 @@
 #define SCB_BASE 0xe000ed00
 #define SCB_CONF_FAULT_STATUS (SCB_BASE + 0x28)
 #define SCB_HARD_FAULT_STATUS  (SCB_BASE + 0x2C)
+#define SCB_MEM_FAULT_ADDRESS  (SCB_BASE + 0x34)
+#define SCB_BUS_FAULT_ADDRESS  (SCB_BASE + 0x38)
 
 static void print_fault_status(void) {
-	uint32_t conf_faults = REG_LOAD(SCB_CONF_FAULT_STATUS);
+	uint32_t conf_faults = REG32_LOAD(SCB_CONF_FAULT_STATUS);
+	uint32_t bus_fault_status, mem_fault_status, usage_fault_status;
 
-	printk("Memory Management Fault Status register = %x\n", conf_faults & 0xf);
-	printk("Bus Fault Status register = %x\n", (conf_faults >> 8) & 0xf);
-	printk("Usage Fault Status register = %x\n", (conf_faults >> 16) & 0xff);
-	printk("Hard Fault Status register = %lx\n", REG_LOAD(SCB_HARD_FAULT_STATUS));
+	mem_fault_status = conf_faults & 0xf;
+	bus_fault_status = (conf_faults >> 8) & 0xf;
+	usage_fault_status = (conf_faults >> 16) & 0xff;
+
+	printk("MemManage Fault Status register = %x\n", mem_fault_status);
```

Comment:
I guess `MemManage Fault Status register` should be `MemManage Fault Status` here (because it's not a register), and the same stuff for following lines.

Also, I think it's better to use `0x%0Nx` format so it's clear that the value is hex.